Core/Battler.gd

```diff
@@ -5,16 +5,19 @@ class_name Battler
 ## This class holds a list of all the enemies, so it's a good central place to dispatch
 ## battle actions (player clicking on enemies, enemy attacks, applying status).
 
-@export var enemies_to_summon: Array[PackedScene]
-@export var enemy_spacing: float = 50.0
+
+@export var enemy_x_spacing: float = 100
+@export var enemy_y_spacing: float = -30
 @export var enemy_attack_delay: float = 0.3
 
 var _enemy_list: Array[Entity]
 var _enemy_action_list: Array[EnemyAction] = []
 
 func _ready() -> void:
-	_summon_enemies()
-	EnemyManager.enemy_list = _enemy_list
+	var enemy_group: EnemyGroup = EnemyManager.choose_enemy_group().instantiate()
+	add_child(enemy_group)
+	_summon_enemies(enemy_group)
+	EnemyManager.current_enemy_group.enemy_list = _enemy_list
```

Comment:
Is there any point in assigning the enemy_list here ?

Reply:
well first of all it's incorrect and I corrected it in 7593c00238bc

And it's to keep track of the current list of enemy, since you need to instantiate the enemy group to get access to the enemy list (which the enemy manager doesn't do)
I think even using the `_enemy_list` of enemy battler is just a way to not have to go search for it inside the `EnemyManager` each time

Reply:
I think you are right, it's mainly an artifact from the time we didn't have the enemy manager and only the battler i think. I'll remove it, it will prevent confusion

Reply:
After checking, it's actually used 11 times, which is starting to be a bit more. Maybe it makes sense to keep it then, instead of replacing everything by `EnemyManager.current_enemy_group.enemy_list`